packages/core/contracts/data-verification-mechanism/implementation/VotingV2.sol

```diff
@@ -718,7 +738,11 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
         emit GatChanged(newGat);
     }
 
-    // Here for abi compatibility. to be removed.
+    /**
+     * @notice Does nothing.
+     * @dev Here for abi compatibility only, to be removed.
```

Comment:
To be removed? What does this mean? Is this deprecated or should it be removed before launch?

Reply:
that's a good point. I think we can actually totally remove this; there is no point in having this interface, assuming it is removed from `VotingV2Interface.sol`. I've removed it from both locations.